esp-hal/src/gpio/lp_io.rs

```diff
@@ -141,48 +156,47 @@ impl<'d, const PIN: u8> LowPowerOutputOpenDrain<'d, PIN> {
     }
 
     fn input_enable(&self, enable: bool) {
-        get_pin_reg(PIN).modify(|_, w| w.fun_ie().bit(enable));
+        let lp_io = unsafe { LP_IO::steal() };
+        lp_io
+            .gpio(PIN as usize)
+            .modify(|_, w| w.fun_ie().bit(enable));
     }
 
     /// Sets pull-up enable for the pin
     pub fn pullup_enable(&self, enable: bool) {
-        get_pin_reg(PIN).modify(|_, w| w.fun_wpu().bit(enable));
```

Comment:
Might've been worth keeping the `get_pin_reg` function still. :sweat_smile:

Reply:
It's impossible if we use `::steal()` because it would return a reference to a local now. We don't have to use `steal`, thoguh, but it looks better to my brain than random pointer dereferencing.

Reply:
> but it looks better to my brain than random pointer dereferencing

Same :smile: